components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -398,10 +405,7 @@ export const StyledWidgetToggle = styled<WidgetToggleProps, 'button'>('button')`
 
 export const StyledButtonIcon = styled<{}, 'div'>('div')`
   display: inline-block;
-  vertical-align: sub;
   margin-right: 5px;
-  height: 20px;
```

Comment:
The svg child element has an explicit width and height but when you remove the parent explicit width and height then the parent element takes more height than the child resulting in the icon vertical alignment being off:

_On left is making it aligned correctly via DevTools, on right is current PR state..._
![image](https://user-images.githubusercontent.com/741836/100645548-add72f80-32f1-11eb-975f-2ac9740c3fb8.png)

Recommend putting explicit icon container width and height and having the svg child be width and height `100%`, or removing the parent element and having the svg be direct child if you don't need to manipulate it (less likely).

Reply:
Should be fixed now, using the `PlusIcon` from `brave-ui`

<img width="205" alt="Screen Shot 2020-12-02 at 9 49 53 PM" src="https://user-images.githubusercontent.com/8732757/100966398-07a24a00-34ea-11eb-9e00-86f7fb2953ac.png">
<img width="207" alt="Screen Shot 2020-12-02 at 9 00 13 PM" src="https://user-images.githubusercontent.com/8732757/100966405-096c0d80-34ea-11eb-8a8f-be0cc9aa80d8.png">